core-tests/jvm/src/test/scala/zio/IOSpec.scala

```diff
@@ -122,19 +122,19 @@ class IOSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRuntim
   def t2 = {
     val list    = List("1", "2", "3")
     val effects = new mutable.ListBuffer[String]
-    val res     = unsafeRun(IO.foreach(list)(x => IO.effectTotal(effects += x) *> IO.succeedLazy[Int](x.toInt)))
+    val res     = unsafeRun(IO.foreach(list)(x => IO.effectTotal(effects += x) *> IO.succeed[Int](x.toInt)))
     (effects.toList, res) must be_===((list, List(1, 2, 3)))
   }
 
   def t3 = {
     val list = List("1", "h", "3")
-    val res  = Try(unsafeRun(IO.foreach(list)(x => IO.succeedLazy[Int](x.toInt))))
+    val res  = Try(unsafeRun(IO.foreach(list)(x => IO.effect[Int](x.toInt))))
```

Comment:
I think all `_.toInt` in this file should become either `effectTotal` or `succeed` consistently (since it's all the same call and there's no difference between them)

Reply:
Actually I did initially try with `succeed`, but the test failed (aparently laziness matters here).

Reply:
In `t3` lazyness matters, so changing all `_.toInt` to `effectTotal`.